canonical-web-and-design/vanilla-framework
# Update what's new page

## Done

- Add new changelog page for v2
- Update what's new page to include all removed components and changes in 3.0
- Added link to changelog from what's new page

Fixes #4217

## QA

- Open [demo](https://vanilla-framework-4219.demos.haus/docs/whats-new)
- Check the table of removed components especially the descriptions. 
- Check link to changelog page works as expected
- Review updated documentation:
  - The whole of the what's new page

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
